frontend/src/components/ui/form.tsx

```diff
@@ -176,6 +178,24 @@ const FormMessage = React.forwardRef<
 });
 FormMessage.displayName = "FormMessage";
 
+const FormMessageTooltip = ({ className }: { className: string }) => {
+  const { error } = useFormField();
+  const body = error && error?.message ? String(error?.message) : null;
+
+  if (!body) {
+    return null;
+  }
+
+  return (
+    <TooltipProvider>
```

Comment:
Not sure if this is the right place for the `Provider`. I was getting an error saying that `Tooltip` must be used in a provider. But come to think of it maybe that's because I was running in storybook